automations/js/src/utils/projects.mjs

```diff
@@ -164,18 +166,23 @@ class Project {
    * @returns {Promise<string>} the ID of the card that was updated
    */
   async setCustomChoiceField(cardId, fieldName, optionName) {
+    this.core.info(
+      `Setting field "${fieldName}" to value "${optionName}" for card "${cardId}".`
+    )
     // Preliminary validation
     if (!this.fields[fieldName]) {
-      throw new Error(`Unknown field name "${fieldName}".`)
+      const msg = `Unknown field name "${fieldName}".`
+      this.core.error(msg)
```

Comment:
Using `core.error` produces neat little error banners in the workflow output. This is a great way to surface problems:

https://github.com/actions/toolkit/tree/main/packages/core#annotations